app/presenters/node_presenter.rb

```diff
@@ -12,7 +12,7 @@ def node_name
   end
 
   def node_slug
-    node_name.to_s.dasherize
+    @node.slug
```

Comment:
It still needs to stick around so that FlowPresenter instances can establish a slug [based off the current_node](https://github.com/alphagov/smart-answers/blob/733ee752f91dec91c7b41e63ba0a3523a35d8232/app/controllers/flow_controller.rb#L15).

I prefixed it with `node_` so that it matches the `node_name` method above. Both of them could drop the prefix but that accessing name on a question presenter (that inherits from this) feels a bit vague and [other delegations wouldn't make sense](https://github.com/alphagov/smart-answers/blob/733ee752f91dec91c7b41e63ba0a3523a35d8232/app/presenters/flow_presenter.rb#L20) so would need a method adding.